lms/static/sass/course/wiki/_wiki.scss

```diff
@@ -1,5 +1,4 @@
-section.wiki {
-  padding-top: 25px;
+.wiki-wrapper section.wiki {
```

Comment:
Note to self: I think I should add shim import to the top of this file:

@import '../../edx-pattern-library-shims/base/variables';